.github/workflows/repotests.yml

```diff
@@ -0,0 +1,28 @@
+name: Repo tests
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-python@v5
+      with:
+        python-version: '3.10'
+    - name: Install blint
+      run: |
+        python -m pip install poetry
+        poetry install
+    - uses: actions/checkout@v4
+      with:
+        repository: juice-shop/juicy-malware
+        path: /home/runner/work/juicy_malware
+    - name: Test
+      run: |
+        poetry run blint -i /home/runner/work/juicy_malware -o /home/runner/work/reports
```

Comment:
Can we add a couple of apks from [f-droid](https://f-droid.org/packages/) and test the sbom commands as well?